keyboards/ploopyco/trackball_thumb/rev1_001/config.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2021 Colin Lam (Ploopy Corporation)
+ * Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// These pins are not broken out, and cannot be used normally.
+// They are set as output and pulled high, by default
+#define UNUSED_PINS \
```

Comment:
This causes a warning about a deprecated define.

Reply:
Can be safely ignored. Most boards don't actually do anything with the unused pins define.  However, because the ploopy devices wire these pins to ground, the code here respects that and uses this define. 

Check the `keyboard_pre_init_kb` function for reference.

Reply:
Yep, I've seen how it works (I ported these changes to `develop` last week, and ran into these warnings), but with this warning present it's something users will have to overlook. Could it be renamed to avoid the warning from QMK?

Reply:
It could be, but it's being deprecated and removed from most boards soon.  Would rather keep it to avoid the hassle of repeated renames.